src/main/java/com/internet/shop/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,199 @@
+package com.internet.shop.dao.jdbc;
+
+import com.internet.shop.dao.interfaces.OrderDao;
+import com.internet.shop.exceptions.DataProcessingException;
+import com.internet.shop.lib.Dao;
+import com.internet.shop.model.Order;
+import com.internet.shop.model.Product;
+import com.internet.shop.util.ConnectionUtil;
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class OrderDaoJdbcImpl implements OrderDao {
+    @Override
+    public List<Order> getUserOrders(Long id) {
+        String query = "SELECT * FROM orders WHERE user_id = ? AND order_deleted = FALSE;";
```

Comment:
```suggestion
        String query = "SELECT * FROM orders WHERE user_id = ? AND = deleted = FALSE;";
```
it's field in table ordes— no need to add order to column name

Reply:
It's the name of column order_deleted, not name of table.

Reply:
![image](https://user-images.githubusercontent.com/48496129/94479149-08aeb680-01dd-11eb-8b8c-1ce0893bf810.png)